react/components/molecules/item-notification/item-notification.js

```diff
@@ -0,0 +1,88 @@
+import React, { PureComponent } from "react";
+import { StyleSheet, Text, TouchableOpacity } from "react-native";
+import PropTypes from "prop-types";
+
+import { Avatar } from "../../atoms/avatar";
+import { timeStringUTC } from "../../../util";
+
+export class ItemNotification extends PureComponent {
+    static get propTypes() {
+        return {
+            avatarURL: PropTypes.String,
+            onPress: PropTypes.func,
+            text: PropTypes.String,
+            timestamp: PropTypes.Number,
+            unread: PropTypes.bool
+        };
+    }
+
+    static get defaultProps() {
+        return {
+            onPress: undefined,
+            text: null,
+            unread: false
+        };
+    }
+
+    _style() {
+        return [
+            styles.style,
+            {
+                backgroundColor: this.props.unread ? "#e5ebfd" : "#fff"
+            }
+        ];
+    }
+
+    render() {
+        return (
+            <TouchableOpacity
+                style={this._style()}
+                onPress={this.props.onPress}
+                activeOpacity={0.6}
+            >
+                <Avatar
+                    image={{
+                        uri: this.props.avatarURL
+                    }}
+                    size={52}
+                    style={styles.avatar}
+                />
+                <Text style={styles.text}>{this.props.text}</Text>
+                {this.props.timestamp ? (
+                    <Text style={styles.timestamp}>{timeStringUTC(this.props.timestamp)} </Text>

```

Comment:
Use this function for the time: https://github.com/ripe-tech/ripe-components-react-native/blob/333e6e95882c55c9760bba43128c7a7c3c17b424/react/util/utils.js#L41

Baically, if it is in the same day it displays the time in hh:mm:ss, if it isn't the same day, it displays the date too